modyn/supervisor/internal/grpc/enums.py

```diff
@@ -18,23 +18,11 @@ class PipelineStatus(StrEnum):
     NOTFOUND = "not found"
 
 
-class PipelineType(StrEnum):
```

Comment:
Decided that a parent relation is more useful for pipeline hierarchy graph/tree plot.